ResourceStatusSystem/Utilities/Synchronizer.py

```diff
@@ -55,6 +56,36 @@ def sync( self, _eventName, _params ):
     :return: S_OK
     '''
 
+    sesHosts = CSHelpers.getStorageElementsHosts()
+    if not sesHosts[ 'OK' ]:
+      return sesHosts
+    sesHosts = sesHosts[ 'Value' ]
+
+    resources = sesHosts
+
+    ftsServer = getFTS3Servers()
+    if ftsServer[ 'OK' ]:
+      resources.extend( ftsServer[ 'Value' ] )
```

Comment:
Actually I made a new private method `__removeNonExistingResources`, tell me what you think